frontend/src/__tests__/cypress/cypress/e2e/modelRegistry/ModelVersionArchive.cy.ts

```diff
@@ -0,0 +1,320 @@
+/* eslint-disable camelcase */
+import { mockK8sResourceList } from '~/__mocks__';
+import { mockDashboardConfig } from '~/__mocks__/mockDashboardConfig';
+import { MODEL_REGISTRY_API_VERSION } from '~/concepts/modelRegistry/const';
+import { mockModelRegistry } from '~/__mocks__/mockModelRegistry';
+import { mockRegisteredModelList } from '~/__mocks__/mockRegisteredModelsList';
+import { ModelRegistryModel } from '~/__tests__/cypress/cypress/utils/models';
+import { mockModelVersionList } from '~/__mocks__/mockModelVersionList';
+import { mockModelVersion } from '~/__mocks__/mockModelVersion';
+import {
+  ModelRegistryMetadataType,
+  ModelVersion,
+  ModelVersionState,
+} from '~/concepts/modelRegistry/types';
+import { mockRegisteredModel } from '~/__mocks__/mockRegisteredModel';
+import { verifyRelativeURL } from '~/__tests__/cypress/cypress/utils/url';
+import {
+  archiveVersionModal,
+  modelVersionArchive,
+  restoreVersionModal,
+} from '~/__tests__/cypress/cypress/pages/modelRegistry/modelVersionArchive';
+import { labelModal, modelRegistry } from '~/__tests__/cypress/cypress/pages/modelRegistry';
+
+type HandlersProps = {
+  registeredModelsSize?: number;
+  modelVersions?: ModelVersion[];
+};
+
+const initIntercepts = ({
+  registeredModelsSize = 4,
+  modelVersions = [
+    mockModelVersion({
+      name: 'model version 1',
+      author: 'Author 1',
+      id: '1',
+      customProperties: {
```

Comment:
Do we need the `customProperties` here ? If yes, can we move this to another shared file since a lot of mocks use this?